package.json

```diff
@@ -78,6 +79,7 @@
     "react-query": "^3.39.3"
   },
   "dependencies": {
+    "lodash": "^4.17.21",
```

Comment:
Nit-pick - if we have a chance to simplify down to `Object.assign` and bring this out later, I'd be in favor of that route.  Nbd in the meantime - nice that we're NOT adding as a peer dep here for now, since it might be temporary